docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -259,55 +281,79 @@ The following command operators are supported:
   TCP
     Not implemented (0).
 
+  UDP
+    Number of downstream datagrams with no healthy host found in the upstream cluster. (reused operator for udp proxy).
```

Comment:
I'm very sorry if I missed this before, but I think the reuse of this formatter is very confusing. I think we have 2 options that we could potentially use:

1) Add generic new formatters that could cover these cases. If these seem very specific to UDP this is probably not a great option.
2) Add a UDP specific metadata and then allow logging through it via the `DYNAMIC_METADATA` formatter. IMO this is the better option as then you can have very UDP specific things and document that clearly along side the UDP proxy filter.

My recommendation is to go with (2) for the UDP specific things. I think it will end up clean and easy to understand. What do you think?

Reply:
Yeah, indeed the current docs looks confusing, and (2) looks more convenient and has good compatibility, I will implement it soon.

> I'm very sorry if I missed this before, but I think the reuse of this formatter is very confusing. I think we have 2 options that we could potentially use:
> 
> 1. Add generic new formatters that could cover these cases. If these seem very specific to UDP this is probably not a great option.
> 2. Add a UDP specific metadata and then allow logging through it via the `DYNAMIC_METADATA` formatter. IMO this is the better option as then you can have very UDP specific things and document that clearly along side the UDP proxy filter.
> 
> My recommendation is to go with (2) for the UDP specific things. I think it will end up clean and easy to understand. What do you think?

Yeah, indeed the current docs looks confusing, and (2) looks more convenient and has good compatibility, I will implement it soon.

Reply:
> I'm very sorry if I missed this before, but I think the reuse of this formatter is very confusing. I think we have 2 options that we could potentially use:
> 
> 1. Add generic new formatters that could cover these cases. If these seem very specific to UDP this is probably not a great option.
> 2. Add a UDP specific metadata and then allow logging through it via the `DYNAMIC_METADATA` formatter. IMO this is the better option as then you can have very UDP specific things and document that clearly along side the UDP proxy filter.
> 
> My recommendation is to go with (2) for the UDP specific things. I think it will end up clean and easy to understand. What do you think?

Yeah, indeed the current docs looks confusing, and (2) looks more convenient and has good compatibility, I will implement it soon.

Reply:
These docs need cleaning up now.